.github/README.md

```diff
@@ -12,12 +12,18 @@
 
 ## Hvordan starte serveren?
 ### Kjøre programmet:
-`dotnet run`
+```console
+$ cd nordicdoor/PDSA_System/Server/ 
```

Comment:
Det er vanskelig å vite hvordan det vil se ut for den som kloner repoet. Kloner de gjennom en IDE så vil de få spørsmål om hva de vil mappen skal hete, og da kan det være mulig at de velger et annet navn enn `nordicdoor`. Samme gjelder også når man oppretter en egen fork, da blir man spurt om hva forken skal hete

Jeg hadde ikke antatt hvordan mappestrukturen til folk ser ut, det eneste vi vet er at de har mappene `Client`, `Server` og `Shared`